### PR TITLE
Removed usage of logger.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Removed usage of the CKEditor 5 logger. Part of ckeditor/ckeditor5#383.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: src/conversion/downcasthelpers.js [hunk 1]
```diff
@@ -824,8 +826,10 @@ function changeAttribute( attributeCreator ) {
 			 *
 			 * @error conversion-attribute-to-attribute-on-text
 			 */
-			log.warn( 'conversion-attribute-to-attribute-on-text: ' +
-				'Trying to convert text node\'s attribute with attribute-to-attribute converter.' );
+			console.warn( attachLinkToDocumentation(
```
Comment (pjasiun) [inline src/conversion/downcasthelpers.js]: This should be `CKEditorError` exception.

File: src/model/utils/insertcontent.js [hunk 2]
```diff
@@ -322,8 +318,8 @@ class Insertion {
 		if ( !this.schema.checkChild( this.position, node ) ) {
 			// Algorithm's correctness check. We should never end up here but it's good to know that we did.
 			// Note that it would often be a silent issue if we insert node in a place where it's not allowed.
-			log.error(
-				'insertcontent-wrong-position: The node cannot be inserted on the given position.',
+			console.error(
+				attachLinkToDocumentation( 'insertcontent-wrong-position: The node cannot be inserted on the given position.' ),
```
Comment (pjasiun) [inline src/model/utils/insertcontent.js]: This should be `CKEditorError` exception.

File: src/model/utils/insertcontent.js [hunk 3]
```diff
@@ -442,7 +438,7 @@ class Insertion {
 				// Algorithm's correctness check. We should never end up here but it's good to know that we did.
 				// At this point the insertion position should be after the node we'll merge. If it isn't,
 				// it should need to be secured as in the left merge case.
-				log.error( 'insertcontent-wrong-position-on-merge: The insertion position should equal the merge position' );
```
Comment (pjasiun) [inline src/model/utils/insertcontent.js]: This should be `CKEditorError` exception.

File: src/model/utils/insertcontent.js [hunk 4]
```diff
@@ -442,7 +438,7 @@ class Insertion {
...
+				// @if CK_DEBUG // console.error( 'The insertion position should equal the merge position' );
```
Comment (pjasiun) [inline src/model/utils/insertcontent.js]: This should be `CKEditorError` exception.

File: src/view/view.js [hunk 5]
```diff
@@ -405,7 +406,9 @@ export default class View {
 				 *
 				 * @error view-focus-no-selection
 				 */
-				log.warn( 'view-focus-no-selection: There is no selection in any editable to focus.' );
+				console.warn( attachLinkToDocumentation(
```
Comment (pjasiun) [inline src/view/view.js]: This should be `// @if CK_DEBUG //`

File: src/conversion/downcasthelpers.js (hunk 1 above)
Comment (ma2ciek) [reply src/conversion/downcasthelpers.js]: OTH it doesn't break the editor.

Comment (pjasiun) [reply src/conversion/downcasthelpers.js]: But this means that someone used our API in an incorrect way. I think there are 2 ways:  we agreed that using an attribute to attribute converter on text is fine and do not warn about it, or we agreed that it is not fine, and does not allow people to use it (throw `CKEditorError`). It does not make much sense to be in between.

Comment (pjasiun) [reply src/conversion/downcasthelpers.js]: I think we should throw now. I will ping @scofalik, if he will know about some cases when it is correct, we will change it later.

File: src/conversion/downcasthelpers.js [hunk 6]
```diff
@@ -3,6 +3,14 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
+/**
+ * Contains downcast (model-to-view) converters for {@link module:engine/conversion/downcastdispatcher~DowncastDispatcher}.
+ *
+ * @module engine/conversion/downcasthelpers
+ */
+
+/* globals */
```
Comment (pjasiun) [inline src/conversion/downcasthelpers.js]: Is it needed?

Comment (scofalik) [reply src/conversion/downcasthelpers.js]:
```suggestion
```

Comment (pjasiun) [reply src/conversion/downcasthelpers.js]:
```suggestion
```

File: src/conversion/downcasthelpers.js [hunk 7]
```diff
@@ -3,6 +3,13 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
+/**
+ * Contains downcast (model-to-view) converters for {@link module:engine/conversion/downcastdispatcher~DowncastDispatcher}.
+ *
+ * @module engine/conversion/downcasthelpers
+ */
+
+
```
Comment (scofalik) [inline src/conversion/downcasthelpers.js]:
```suggestion
```

**Pull request merged**